html-forms/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form>
+    <label for="email">Email</label>
+    <div>
+      <input type="email">
+    </div>
+
+    <label for="options">Why is coding cool?</label>
+    <div>
+      <select>
+        <option>Select an option</option>
+        <option>Because it's interesting</option>
+        <option>Because it's fun</option>
+        <option>Because it's powerful</option>
+      </select>
+    </div>
+
+    <label for="languages">Favorite Language (choose one)</label>
```

Comment:
This can just be a general text element such as `p`, it isn't labelling a specific input.